lib/uplinkc/testdata_test.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testplanet"
+)
+
+func TestC(t *testing.T) {
+	ctx := testcontext.New(t)
+	//	defer ctx.Cleanup()
```

Comment:
Keep the cleanup, might have been my mistake.

Reply:
:+1: